### ISSUE TITLE
support macOS12?

### ISSUE THREAD

**Issue by 368573607** [microsoft/playwright#10457]
I tried to install playwright on the monterey system, but an error occurred. I don't know if this is a problem with my system. The error is as follows:
```
error /Users/***/Documents/GitHub/from-cpp-web/node_modules/playwright: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /Users/***/Documents/GitHub/from-cpp-web/node_modules/playwright
Output:
/Users/***/Documents/GitHub/from-cpp-web/node_modules/proper-lockfile/lib/lockfile.js:68
                return callback(Object.assign(new Error('Lock file is already being held'), { code: 'ELOCKED', file }));
                                              ^

Error: Lock file is already being held
    at /Users/***/Documents/GitHub/from-cpp-web/node_modules/proper-lockfile/lib/lockfile.js:68:47
    at callback (/Users/***/Documents/GitHub/from-cpp-web/node_modules/graceful-fs/polyfills.js:299:20)
    at FSReqCallback.oncomplete (node:fs:199:5) {
  code: 'ELOCKED',
  file: '/Users/***/Library/Caches/ms-playwright'
}

Node.js v17.0.1
```
I noticed that registry.ts contains this code:

![image](https://user-images.githubusercontent.com/60562136/142750226-804322c4-4d9b-4b7d-af5c-8b54da8b3ab1.png)

The above does not seem to contain mac12, but I am not sure.

![image](https://user-images.githubusercontent.com/60562136/142750007-65407fae-6179-4c41-b0a1-0e1145fa9e39.png)

**Comment by mxschmitt** [comment]
Which Playwright version are you using? MacOS 12 is supported.

It might get fixed by https://github.com/microsoft/playwright/pull/10356 already, so using `npm i playwright@next` should include the fix.

**Comment by 368573607** [comment]
> It might get fixed by #10356 already, so using `npm i playwright@next` should include the fix.

Thanks:smile:
